.github/workflows/stable-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Locked Dependencies
+on:
+    push:
+    pull_request:
+    schedule:
+        -   cron: '0 */12 * * *'
```

Comment:
running them twice a day in addition to running them on every push seems excessive